<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComicViewer</class>
 <widget class="QWidget" name="ComicViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonInfo">
       <property name="toolTip">
        <string>View comic information</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ComicDesktop.qrc">
         <normaloff>:/ComicDesktop/Resources/Information.png</normaloff>:/ComicDesktop/Resources/Information.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonBrowser">
       <property name="toolTip">
        <string>Browse the pages of the comic</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ComicDesktop.qrc">
         <normaloff>:/ComicDesktop/Resources/Browse.png</normaloff>:/ComicDesktop/Resources/Browse.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton1">
       <property name="toolTip">
        <string>Show one page at a time</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ComicDesktop.qrc">
         <normaloff>:/ComicDesktop/Resources/1.png</normaloff>:/ComicDesktop/Resources/1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton2">
       <property name="toolTip">
        <string>Show two pages at a time</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ComicDesktop.qrc">
         <normaloff>:/ComicDesktop/Resources/2.png</normaloff>:/ComicDesktop/Resources/2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonMagnify">
       <property name="toolTip">
        <string>Magnify the highlighted area with the mouse</string>
       </property>
       <property name="text">
        <string>Magnify</string>
       </property>
       <property name="icon">
        <iconset resource="ComicDesktop.qrc">
         <normaloff>:/ComicDesktop/Resources/Magnify.png</normaloff>:/ComicDesktop/Resources/Magnify.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonRotateLeft">
       <property name="toolTip">
        <string>Rotate image counterclockwise (single page view only)</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ComicDesktop.qrc">
         <normaloff>:/ComicDesktop/Resources/RotateLeft.png</normaloff>:/ComicDesktop/Resources/RotateLeft.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonRotateRight">
       <property name="toolTip">
        <string>Rotate image clockwise (single page view only)</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ComicDesktop.qrc">
         <normaloff>:/ComicDesktop/Resources/RotateRight.png</normaloff>:/ComicDesktop/Resources/RotateRight.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label4x">
       <property name="text">
        <string>4X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sliderZoom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="singleStep">
        <number>25</number>
       </property>
       <property name="pageStep">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label1x">
       <property name="text">
        <string>1X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ComicView" name="graphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="dragMode">
      <enum>QGraphicsView::ScrollHandDrag</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BrowserContent" name="widgetBrowser" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComicView</class>
   <extends>QGraphicsView</extends>
   <header>ComicView.h</header>
  </customwidget>
  <customwidget>
   <class>BrowserContent</class>
   <extends>QWidget</extends>
   <header>BrowserContent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>toolButtonBrowser</tabstop>
  <tabstop>toolButton1</tabstop>
  <tabstop>toolButton2</tabstop>
  <tabstop>sliderZoom</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources>
  <include location="ComicDesktop.qrc"/>
 </resources>
 <connections/>
</ui>
